src/sage/schemes/elliptic_curves/constructor.py

```diff
@@ -146,7 +146,8 @@ class EllipticCurveFactory(UniqueFactory):
 
         sage: E = EllipticCurve(CC, [0,0,1,-1,0])
         sage: E
-        Elliptic Curve defined by y^2 + 1.00000000000000*y = x^3 + (-1.00000000000000)*x over Complex Field with 53 bits of precision
+        Elliptic Curve defined by y^2 + 1.00000000000000*y = x^3 + (-1.00000000000000)*x
+        over Complex Field with 53 bits of precision
```

Comment:
Sometimes I see
```
Elliptic Curve ...
 over ...
```
Other times
```
Elliptic Curves ...
over ...
```
Need to be consistent. Here and elsewhere.

Reply:
I am standardizing on the first format now

Reply:
Okay for this PR.

But for standardizing how to split a long doctest output and formatting nicely multiline output, I don't know what is right. A related idea is to respect the actual output in the sage terminal. I don't know whether we should or not. This issue perhaps belongs to https://doc.sagemath.org/html/en/developer/coding_basics.html#fine-points-on-styles

Reply:
I don't think there's a need for setting a policy for developers; to me, anything in the spectrum between the raw actual output (assisted by `sage -fixdoctests`) and the delicate doctest poetry that I have attempted here is fine. I don't think there is a downside to reformatting the output with the goal of readability / user experience. There is potential for mild disappointment when users try out an example and the output does not look as pretty as it appears in the documentation. But even then the structure-indicating reformatted output guides the user through parsing the complicated raw output. And Sage has various ways to adjust output, using `%display latex`, `%display ascii_art` etc.; the difference between documentation and reality may inspire users to explore these facilities (or even improve our `_repr_` code).